doc/source/features/online-schema-changes.rst

```diff
@@ -0,0 +1,82 @@
+.. _online-schema-changes:
+
+===================================================
+Online Schema Changes
+===================================================
+
+A failure in a schema change can cause your cluster to drop nodes and lose data. |Percona XtraDB Cluster| supports the following methods for making online schema changes:
+
+.. list-table::
+    :widths: 20 60
+    :header-rows: 1
+
+    * - Method 
+      - Description
+    * - `TOI`
+      - Total Order Isolation
+        
+        The default DDL replication method for the :ref:`wsrep osu method` variable. The isolation of the DDL statement guarantees consistency.
+        
+        The node detects a DDL statement in the Parser stage and sends the statement to all nodes. All nodes wait for any earlier transactions to commit, and then execute the statement.
+        
+        All other writes must wait until the DDL statement is executed. While a DDL statement is running, any long-running transactions in progress and using the same resource receive a deadlock error at commit and are rolled back. There is no way to verify if individual nodes have succeeded in processing the DDL statement. 
+    * - `RSU`
+      - Rolling  Schema Upgrade
+
+        A DDL statement are processed on each nod locally. This action guarantees high availability. The node desynchronizes with the cluster and disables flow control while executing the DDL statement. The rest of the cluster is not affected. After the statement execution, the node applies delayed events and synchronizes with the cluster. 
+
+        The cluster continues to operate, although during the process, some nodes may be updated and some nodes may have the older schema structure.
+
+        For this method, the ``gcache`` must be large enough to store the data for the duration of the DDL change. 
+    * - `NBO`
+      - Non-Blocking Operation
+
+        This method is similar to ``TOI``. All nodes acquire metadata locks on each table or schema for any open transactions and do not release those locks until the transaction ends. The DDL statements execute on all nodes after the transactions have completed. This method avoids the TOI issue when a long-running DDL blocks other updates in the cluster.
+
+        For this method, the ``gcache`` must be large enough to store the data for duration of the DDL statement. 
+
+Using the NBO method
+----------------------
+
+You set the method for online schema changes by either using the ``wsrep_OSU_method`` variable in ``my.cnf`` or  using the ``SET`` statement in a MySQL client. 
+
+To set the variable in the my.cnf file, do the following:
+
+.. sourcecode:: text
+
+    wsrep_osu_method=NBO
+
+Restart the server. 
+
+To dynamically set the ``NBO`` method in the client, run the following statement:
+
+.. sourcecode:: mysql
+
+    SET GLOBAL wsrep_OSU_method='NBO';
+
+
+Limitations
```

Comment:
Most of the limitations mentioned here are true for all methods.

SInce the page is called "online schema changes", not "NBO online schema change", why do we say that these are NBO limitations?

I also don't see the main NBO limitation mentioned: that it only supports some DDL statements, and that it doesn't support running two ALTER on the same table (or more specifically: two DDLs with conflicting locks) at the same time, one of them will error out.

Reply:
Added.